.github/workflows/template-deploy.yaml

```diff
@@ -0,0 +1,73 @@
+name: Production Subgraph (Mainnet)
```

Comment:
I don't think production is limited to mainnet. In the context of this repo, testnets like goerli, rinkeby, etc. are production. This is also used for mainnet staging, so I think this name could lead to confusion.

Reply:
which one do you recommend? should I use `Production Mainnet` ? it was like this before this PR

Reply:
How about this?
```suggestion
name: Deploy Subgraph (${{ inputs.ENVIRONMENT }})
```

Reply:
this doesn't work like that, the variables are only known later, so the name can't be dynamically created

Reply:
Ah sorry. Just `Deploy Subgraph` then

Reply:
@Theodus please check my latest commit